src/activities/assignments/AssignmentEntity.js

```diff
@@ -894,6 +927,11 @@ export class AssignmentEntity extends Entity {
 			fields.push({ name: 'allowableFileType', value: assignment.allowableFileType });
 		}
 
+		const allowableFileTypeCustomValue = '5';
```

Comment:
Would be best if we didn't have to use a magic number. Although, I don't know the entity enough to suggest another way of getting the value, but if you can think of one, it would be better.

Reply:
We need to use the id/number because first the selection needs to be unique and second because the name (text) will be translated so we cannot compare the selection against a hardcoded value